src/app.ts

```diff
@@ -86,7 +90,10 @@ async function app() {
   const dependencies = program.dependency?.map((dep: string) => dep.trim());
 
   // Load FhirProcessor and config object
-  const processor = getFhirProcessor(inDir, defs);
+  const fileType = ['json-only', 'xml-only', 'json-and-xml'].includes(program.fileType)
+    ? program.fileType
+    : 'json-only';
```

Comment:
I would consider making this case-insensitive:
```ts
/^(json-only|xml-only|json-and-xml)$/i.test(program.fileType)
```

In addition, I think that if the user enters an supported value (e.g., `XML`), we should error out.  Otherwise the output they get may not be the output they expect.